lib/davinci_pas_test_kit/generated/v2.0.1/server_suite.rb

```diff
@@ -79,6 +80,24 @@ class ServerSuite < Inferno::TestSuite
       #   request.query_parameters['token']
       # end
 
+      group 'Subscription Setup' do
+        description %(
+          The Subscription Setup tests verify that the server supports creation of a rest-hook Subscription. The
+          Subscription instance created in these tests will be used for a notification in the pended workflow tests
+          later.
+        )
+        config inputs: { url: { name: :server_endpoint } }
+        input_order :server_endpoint, :smart_credentials, :access_token, :subscription_resource
+        verifies_requirements 'hl7.fhir.us.davinci-pas_2.0.1@141'
+
+        test from: :subscriptions_r4_server_subscription_conformance
+        test from: :pas_server_subscription_input_conformance
+        test from: :subscriptions_r4_server_notification_delivery
```

Comment:
I think we should update the title and description of this test to make clear that the tests are only looking for and verifying the handshake at this time.  They allow notifications as well, but it isn't a focus.

Reply:
https://github.com/inferno-framework/davinci-pas-test-kit/pull/31/commits/439df49a8aefb04f04d53630dbc1f54dc05459d3